tests/ggml-qnn/run-ggml-qnn.sh

```diff
@@ -27,4 +27,4 @@ else
 fi
 
 adb shell chmod +x /data/local/tmp/${GGML_QNN_TEST}
-adb shell /data/local/tmp/${GGML_QNN_TEST}
+adb shell /data/local/tmp/${GGML_QNN_TEST} perf
```

Comment:
spend a lot of time to investigate why UT of QNN backend with test-backend-ops.cpp not works fine as expected, finally just need a command line param.

Reply:
![Screenshot from 2024-04-26 21-17-30](https://github.com/ggerganov/llama.cpp/assets/6889919/4079d5eb-bc88-41a8-bf3f-ea2ac4031c08)

the test-backend-ops.cpp works fine with QNN backend.

but I can not lie, so I add this param "perf" to verify whether the data path is actually work.